src/app/lib/streamer.js

```diff
@@ -244,8 +244,7 @@
 
     var Streamer = {
         start: function (model) {
-            var torrent = model.get('torrent');
-            var torrentUrl = torrent.url ? torrent.url : torrent.magnet;
+            var torrentUrl = model.get('torrent');
```

Comment:
I haven't, sorry. I just tested drag&dropping a torrent and a magnet link and both worked.